### ISSUE TITLE
Possible bug with AntiforgeryTokenInput.razor component

### ISSUE THREAD

**Issue by VictorioBerra** [damienbod/Blazor.BFF.OpenIDConnect.Template#47]
This component created a hidden input with the SAME NAME as the _Host antiforgery input.

https://github.com/damienbod/Blazor.BFF.OpenIDConnect.Template/blob/main/BlazorBffOpenIdConnect/Client/Shared/AntiForgeryTokenInput.razor#L4

https://github.com/damienbod/Blazor.BFF.OpenIDConnect.Template/blob/main/BlazorBffOpenIdConnect/Server/Pages/_Host.cshtml#L45

You can see the JS file is just going to take the first one: https://github.com/damienbod/Blazor.BFF.OpenIDConnect.Template/blob/main/BlazorBffOpenIdConnect/Client/wwwroot/antiForgeryToken.js#L4

I ran into a problem today because my page is rendering FIRST so the value of the input in `AntiForgeryTokenInput.razor` is null.....

I end up sending a null token to my API and I get this error:

```
[13:55:53 ERR] An unhandled exception has occurred while executing the request.
Microsoft.AspNetCore.Antiforgery.AntiforgeryValidationException: The antiforgery token could not be decrypted.
 ---> System.Security.Cryptography.CryptographicException: The provided payload cannot be decrypted because it was not protected with this protection provider. For more information go to http://aka.ms/dataprotectionwarning
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingBasedDataProtector.UnprotectCore(Byte[] protectedData, Boolean allowOperationsOnRevokedKeys, UnprotectStatus& status)
   at Microsoft.AspNetCore.DataProtection.KeyManagement.KeyRingBasedDataProtector.Unprotect(Byte[] protectedData)
   at Microsoft.AspNetCore.Antiforgery.DefaultAntiforgeryTokenSerializer.Deserialize(String serializedToken)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Antiforgery.DefaultAntiforgeryTokenSerializer.Deserialize(String serializedToken)
   at Microsoft.AspNetCore.Antiforgery.DefaultAntiforgery.DeserializeTokens(HttpContext httpContext, AntiforgeryTokenSet antiforgeryTokenSet, AntiforgeryToken& cookieToken, AntiforgeryToken& requestToken)
   at Microsoft.AspNetCore.Antiforgery.DefaultAntiforgery.ValidateTokens(HttpContext httpContext, AntiforgeryTokenSet antiforgeryTokenSet)
   at Microsoft.AspNetCore.Antiforgery.DefaultAntiforgery.ValidateRequestAsync(HttpContext httpContext)
   at CompromisedAccounts.Web.PreProcessors.AntiforgeryPreProcessor`1.PreProcessAsync(TRequest req, HttpContext ctx, List`1 failures, CancellationToken ct)
```

Proposed fix is maybe update the JS to only send the token that has a value?

IE:

```javascript
function getAntiForgeryToken() {
    var elements = document.getElementsByName('__RequestVerificationToken');

    // https://github.com/damienbod/Blazor.BFF.OpenIDConnect.Template/issues/47
    elements = Array.prototype.slice.call(elements).sort(function (a, b) {
      return b.value.length - a.value.length;
    });

    if (elements.length > 0) {
        return elements[0].value
    }

    console.warn('no anti forgery token found!');
    return null;
}
```

**Comment by damienbod** [comment]
Sounds good, can you make a PR? Greetings Damien

**Comment by damienbod** [comment]
fixed 2.0.4
